src/benchmark/trulens/benchmark/benchmark_frameworks/experiments/rag_triad_baselines.ipynb

```diff
@@ -23,241 +23,29 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.                        latencies.append(end_time - start_time)
</pre><p>why do you have to compute latency here? Don't we track this anyways?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
this was added before the latency fix came in. keeping it here for reproducibility.